### PR TITLE
Paid services

### PR DESCRIPTION
**Pull request by ntorbinskiy**
-Add new page

### CODE REVIEW

**Comment by ntorbinskiy** [comment]
[Paid Services Page](https://trello.com/c/UDvlXxXK/143-paid-services-page)

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -15,6 +16,49 @@ export type Search = {
   posts: PostDocumentWithoutBody[];
 };
 
+interface Page {
+  name: string;
+  link: string;
+}
+
+interface LinksProps {
```
Comment (1alexvash) [inline src/components/Header/index.tsx:24]: Just `Props`

File: helpers/getEdgeNodesHelpers.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { PostDocumentWithoutBody } from "interfaces";
 
+import { pagesName } from "../src/components/Header";
```
Comment (1alexvash) [inline helpers/getEdgeNodesHelpers.ts:3]: I'm not sure this is the best collective name

File: src/pages/index.tsx [hunk 3]
```diff
@@ -171,9 +172,7 @@ export const getStaticProps: GetStaticProps = async () => {
 
   return {
     props: {
-      posts: posts.data.postConnection.edges
-        ?.map((edge) => edge?.node)
-        .reverse(),
+      posts: convertTypesAndGetEdges(posts),
```
Comment (1alexvash) [inline src/pages/index.tsx:175]: Can you add similar logic to the statistics page, so it does not Tina pages as content?

Comment (ntorbinskiy) [reply src/pages/index.tsx:175]: image
I have already done it in previous PR :) 
https://github.com/1alexvash/AlexCodeBlog/pull/167/files#diff-69659084985253a79b905760da9916f24601e74f38b847e577e7821e3b7aa047

**Pull request merged**